guest-tools/image/create-td-image.sh

```diff
@@ -138,16 +143,8 @@ create_guest_image() {
     else
         error "Failed to copy ${CLOUD_IMG} to /tmp"
     fi
-}
 
-config_guest_env() {
```

Comment:
For me personally, it is useful, I don't need to setup some env variable on guest again.

Reply:
for networking access through enterprise proxy ?

Reply:
Yes, but I agree with you, just remove it now, LGTM.